2017-12-03T21:00:46.131+0900 I CONTROL  [initandlisten] MongoDB starting : pid=9585 port=47019 dbpath=/data/shard1/rs2 64-bit host=kris-ThinkPad-X1-Carbon-5th
2017-12-03T21:00:46.131+0900 I CONTROL  [initandlisten] db version v3.4.10
2017-12-03T21:00:46.131+0900 I CONTROL  [initandlisten] git version: 078f28920cb24de0dd479b5ea6c66c644f6326e9
2017-12-03T21:00:46.131+0900 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016
2017-12-03T21:00:46.132+0900 I CONTROL  [initandlisten] allocator: tcmalloc
2017-12-03T21:00:46.132+0900 I CONTROL  [initandlisten] modules: none
2017-12-03T21:00:46.132+0900 I CONTROL  [initandlisten] build environment:
2017-12-03T21:00:46.132+0900 I CONTROL  [initandlisten]     distmod: ubuntu1604
2017-12-03T21:00:46.132+0900 I CONTROL  [initandlisten]     distarch: x86_64
2017-12-03T21:00:46.132+0900 I CONTROL  [initandlisten]     target_arch: x86_64
2017-12-03T21:00:46.132+0900 I CONTROL  [initandlisten] options: { net: { port: 47019 }, processManagement: { fork: true }, replication: { replSet: "s1" }, sharding: { clusterRole: "shardsvr" }, storage: { dbPath: "/data/shard1/rs2" }, systemLog: { destination: "file", path: "s1-r2.log" } }
2017-12-03T21:00:46.153+0900 I STORAGE  [initandlisten] 
2017-12-03T21:00:46.153+0900 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
2017-12-03T21:00:46.153+0900 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
2017-12-03T21:00:46.153+0900 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7269M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),checkpoint=(wait=60,log_size=2GB),statistics_log=(wait=0),
2017-12-03T21:00:46.224+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:46.224+0900 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2017-12-03T21:00:46.224+0900 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2017-12-03T21:00:46.224+0900 I CONTROL  [initandlisten] ** WARNING: You are running this process as the root user, which is not recommended.
2017-12-03T21:00:46.224+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:46.225+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:46.225+0900 I CONTROL  [initandlisten] ** WARNING: /sys/kernel/mm/transparent_hugepage/enabled is 'always'.
2017-12-03T21:00:46.225+0900 I CONTROL  [initandlisten] **        We suggest setting it to 'never'
2017-12-03T21:00:46.225+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:46.225+0900 W SHARDING [initandlisten] Started with --shardsvr, but no shardIdentity document was found on disk in admin.system.version. This most likely means this server has not yet been added to a sharded cluster.
2017-12-03T21:00:46.249+0900 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/shard1/rs2/diagnostic.data'
2017-12-03T21:00:46.270+0900 I REPL     [initandlisten] Did not find local voted for document at startup.
2017-12-03T21:00:46.271+0900 I REPL     [initandlisten] Did not find local replica set configuration document at startup;  NoMatchingDocument: Did not find replica set configuration document in local.system.replset
2017-12-03T21:00:46.271+0900 I NETWORK  [thread1] waiting for connections on port 47019
2017-12-03T21:00:51.354+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37972 #1 (1 connection now open)
2017-12-03T21:00:51.354+0900 I -        [conn1] end connection 127.0.0.1:37972 (1 connection now open)
2017-12-03T21:00:51.354+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37976 #2 (1 connection now open)
2017-12-03T21:00:51.354+0900 I NETWORK  [conn2] received client metadata from 127.0.0.1:37976 conn2: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:00:51.355+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:47017
2017-12-03T21:00:51.356+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:47017, took 1ms (1 connections now open to localhost:47017)
2017-12-03T21:00:53.357+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37990 #3 (2 connections now open)
2017-12-03T21:00:53.357+0900 I -        [conn3] end connection 127.0.0.1:37990 (2 connections now open)
2017-12-03T21:00:53.397+0900 I REPL     [replExecDBWorker-0] Starting replication storage threads
2017-12-03T21:00:53.471+0900 I REPL     [replication-0] Starting initial sync (attempt 1 of 10)
2017-12-03T21:00:53.471+0900 I REPL     [ReplicationExecutor] New replica set config in use: { _id: "s1", version: 1, protocolVersion: 1, members: [ { _id: 0, host: "localhost:47017", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 }, { _id: 1, host: "localhost:47018", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 }, { _id: 2, host: "localhost:47019", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 } ], settings: { chainingAllowed: true, heartbeatIntervalMillis: 2000, heartbeatTimeoutSecs: 10, electionTimeoutMillis: 10000, catchUpTimeoutMillis: 60000, getLastErrorModes: {}, getLastErrorDefaults: { w: 1, wtimeout: 0 }, replicaSetId: ObjectId('5a23e7737da32b8ce5ce759f') } }
2017-12-03T21:00:53.471+0900 I REPL     [ReplicationExecutor] This node is localhost:47019 in the config
2017-12-03T21:00:53.471+0900 I REPL     [ReplicationExecutor] transition to STARTUP2
2017-12-03T21:00:53.472+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:47018
2017-12-03T21:00:53.472+0900 I REPL     [ReplicationExecutor] Member localhost:47017 is now in state SECONDARY
2017-12-03T21:00:53.473+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:47018, took 1ms (1 connections now open to localhost:47018)
2017-12-03T21:00:53.474+0900 I REPL     [ReplicationExecutor] Member localhost:47018 is now in state STARTUP
2017-12-03T21:00:53.475+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37994 #4 (2 connections now open)
2017-12-03T21:00:53.475+0900 I NETWORK  [conn4] received client metadata from 127.0.0.1:37994 conn4: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:00:53.508+0900 I REPL     [replication-0] sync source candidate: localhost:47017
2017-12-03T21:00:53.508+0900 I STORAGE  [replication-0] dropAllDatabasesExceptLocal 1
2017-12-03T21:00:53.508+0900 I REPL     [replication-0] ******
2017-12-03T21:00:53.509+0900 I REPL     [replication-0] creating replication oplog of size: 6116MB...
2017-12-03T21:00:53.529+0900 I STORAGE  [replication-0] Starting WiredTigerRecordStoreThread local.oplog.rs
2017-12-03T21:00:53.529+0900 I STORAGE  [replication-0] The size storer reports that the oplog contains 0 records totaling to 0 bytes
2017-12-03T21:00:53.529+0900 I STORAGE  [replication-0] Scanning the oplog to determine where to place markers for truncation
2017-12-03T21:00:53.629+0900 I REPL     [replication-0] ******
2017-12-03T21:00:53.630+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:00:53.631+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (1 connections now open to localhost:47017)
2017-12-03T21:00:53.633+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:00:53.635+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (2 connections now open to localhost:47017)
2017-12-03T21:00:53.636+0900 I REPL     [replication-1] No need to apply operations. (currently at { : Timestamp 1512302451000|1 })
2017-12-03T21:00:53.637+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending connection to host localhost:47017 due to bad connection status; 1 connections to that host remain open
2017-12-03T21:00:53.637+0900 I REPL     [replication-1] Finished fetching oplog during initial sync: CallbackCanceled: Callback canceled. Last fetched optime and hash: { ts: Timestamp 1512302451000|1, t: -1 }[2865188406503243212]
2017-12-03T21:00:53.637+0900 I REPL     [replication-1] Initial sync attempt finishing up.
2017-12-03T21:00:53.637+0900 I REPL     [replication-1] Initial Sync Attempt Statistics: { failedInitialSyncAttempts: 0, maxFailedInitialSyncAttempts: 10, initialSyncStart: new Date(1512302453471), initialSyncAttempts: [], fetchedMissingDocs: 0, appliedOps: 0, initialSyncOplogStart: Timestamp 1512302451000|1, initialSyncOplogEnd: Timestamp 1512302451000|1, databases: { databasesCloned: 0 } }
2017-12-03T21:00:53.645+0900 I REPL     [replication-1] initial sync done; took 0s.
2017-12-03T21:00:53.645+0900 I REPL     [replication-1] Starting replication fetcher thread
2017-12-03T21:00:53.645+0900 I REPL     [replication-1] Starting replication applier thread
2017-12-03T21:00:53.645+0900 I REPL     [replication-1] Starting replication reporter thread
2017-12-03T21:00:53.646+0900 I REPL     [rsSync] transition to RECOVERING
2017-12-03T21:00:53.646+0900 I REPL     [rsBackgroundSync] could not find member to sync from
2017-12-03T21:00:53.649+0900 I REPL     [ReplicationExecutor] Member localhost:47018 is now in state STARTUP2
2017-12-03T21:00:53.649+0900 I REPL     [rsSync] transition to SECONDARY
2017-12-03T21:00:58.650+0900 I REPL     [ReplicationExecutor] Member localhost:47018 is now in state SECONDARY
2017-12-03T21:01:03.651+0900 I REPL     [ReplicationExecutor] Member localhost:47017 is now in state PRIMARY
2017-12-03T21:01:08.653+0900 I REPL     [rsBackgroundSync] sync source candidate: localhost:47017
2017-12-03T21:01:08.654+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:01:08.655+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (2 connections now open to localhost:47017)
2017-12-03T21:01:08.661+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38094 #5 (3 connections now open)
2017-12-03T21:01:08.661+0900 I NETWORK  [conn5] received client metadata from 127.0.0.1:38094 conn5: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:01:08.662+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38096 #6 (4 connections now open)
2017-12-03T21:01:08.663+0900 I NETWORK  [conn6] received client metadata from 127.0.0.1:38096 conn6: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.738+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38170 #7 (5 connections now open)
2017-12-03T21:02:10.738+0900 I NETWORK  [conn7] received client metadata from 127.0.0.1:38170 conn7: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.799+0900 I INDEX    [repl writer worker 1] build index on: admin.system.version properties: { v: 2, key: { version: 1 }, name: "incompatible_with_version_32", ns: "admin.system.version" }
2017-12-03T21:02:10.799+0900 I INDEX    [repl writer worker 1] 	 building index using bulk method; build may temporarily use up to 500 megabytes of RAM
2017-12-03T21:02:10.801+0900 I INDEX    [repl writer worker 1] build index done.  scanned 0 total records. 0 secs
2017-12-03T21:02:10.803+0900 I COMMAND  [repl writer worker 2] setting featureCompatibilityVersion to 3.4
2017-12-03T21:02:10.809+0900 I SHARDING [repl writer worker 13] initializing sharding state with: { _id: "shardIdentity", configsvrConnectionString: "csReplSet/localhost:57040,localhost:57041,localhost:57042", shardName: "s1", clusterId: ObjectId('5a23e786bf51d32cbcc946a4') }
2017-12-03T21:02:10.809+0900 I SHARDING [repl writer worker 13] first cluster operation detected, adding sharding hook to enable versioning and authentication to remote servers
2017-12-03T21:02:10.809+0900 I NETWORK  [repl writer worker 13] Starting new replica set monitor for csReplSet/localhost:57040,localhost:57041,localhost:57042
2017-12-03T21:02:10.810+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57040 (1 connections now open to localhost:57040 with a 5 second timeout)
2017-12-03T21:02:10.812+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57042 (1 connections now open to localhost:57042 with a 5 second timeout)
2017-12-03T21:02:10.813+0900 I SHARDING [thread2] creating distributed lock ping thread for process kris-ThinkPad-X1-Carbon-5th:47019:1512302530:-41758715004048657 (sleeping for 30000ms)
2017-12-03T21:02:10.813+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-03T21:02:10.813+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-03T21:02:10.814+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57041 (1 connections now open to localhost:57041 with a 5 second timeout)
2017-12-03T21:02:10.814+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (2 connections now open to localhost:57040)
2017-12-03T21:02:10.816+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 3ms (2 connections now open to localhost:57040)
2017-12-03T21:02:10.819+0900 I NETWORK  [shard registry reload] Starting new replica set monitor for s0/localhost:37017,localhost:37018,localhost:37019
2017-12-03T21:02:10.819+0900 I NETWORK  [shard registry reload] Starting new replica set monitor for s1/localhost:47017,localhost:47018,localhost:47019
2017-12-03T21:02:10.820+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37017 (1 connections now open to localhost:37017 with a 5 second timeout)
2017-12-03T21:02:10.821+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37019 (1 connections now open to localhost:37019 with a 5 second timeout)
2017-12-03T21:02:10.822+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37018 (1 connections now open to localhost:37018 with a 5 second timeout)
2017-12-03T21:02:10.823+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47017 (1 connections now open to localhost:47017 with a 5 second timeout)
2017-12-03T21:02:10.827+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: LockStateChangeFailed: findAndModify query predicate didn't match any lock document
2017-12-03T21:02:10.847+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38208 #8 (6 connections now open)
2017-12-03T21:02:10.847+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38236 #9 (7 connections now open)
2017-12-03T21:02:10.849+0900 I NETWORK  [conn8] received client metadata from 127.0.0.1:38208 conn8: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.851+0900 I NETWORK  [conn9] received client metadata from 127.0.0.1:38236 conn9: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.853+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47019 (1 connections now open to localhost:47019 with a 5 second timeout)
2017-12-03T21:02:10.857+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47018 (1 connections now open to localhost:47018 with a 5 second timeout)
2017-12-03T21:02:10.867+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38246 #10 (8 connections now open)
2017-12-03T21:02:10.868+0900 I NETWORK  [conn10] received client metadata from 127.0.0.1:38246 conn10: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.880+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38252 #11 (9 connections now open)
2017-12-03T21:02:10.881+0900 I NETWORK  [conn11] received client metadata from 127.0.0.1:38252 conn11: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.930+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38304 #12 (10 connections now open)
2017-12-03T21:02:10.930+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38306 #13 (11 connections now open)
2017-12-03T21:02:10.930+0900 I NETWORK  [conn13] received client metadata from 127.0.0.1:38306 conn13: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.930+0900 I NETWORK  [conn12] received client metadata from 127.0.0.1:38304 conn12: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.992+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38330 #14 (12 connections now open)
2017-12-03T21:02:10.992+0900 I NETWORK  [conn14] received client metadata from 127.0.0.1:38330 conn14: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:11.204+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38352 #15 (13 connections now open)
2017-12-03T21:02:11.204+0900 I NETWORK  [conn15] received client metadata from 127.0.0.1:38352 conn15: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:27.451+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38376 #16 (14 connections now open)
2017-12-03T21:02:27.451+0900 I NETWORK  [conn16] received client metadata from 127.0.0.1:38376 conn16: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:27.607+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38394 #17 (15 connections now open)
2017-12-03T21:02:27.607+0900 I NETWORK  [conn17] received client metadata from 127.0.0.1:38394 conn17: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:40.678+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38428 #18 (16 connections now open)
2017-12-03T21:02:40.679+0900 I NETWORK  [conn18] received client metadata from 127.0.0.1:38428 conn18: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:40.682+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:38434 #19 (17 connections now open)
2017-12-03T21:02:40.682+0900 I NETWORK  [conn19] received client metadata from 127.0.0.1:38434 conn19: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:40.820+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T21:02:40.821+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-03T21:02:40.822+0900 I NETWORK  [shard registry reload] Starting new replica set monitor for s2/localhost:57017,localhost:57018,localhost:57019
2017-12-03T21:02:40.825+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57017 (1 connections now open to localhost:57017 with a 5 second timeout)
2017-12-03T21:02:40.828+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57018 (1 connections now open to localhost:57018 with a 5 second timeout)
2017-12-03T21:02:40.830+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57019 (1 connections now open to localhost:57019 with a 5 second timeout)
2017-12-03T21:03:10.817+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Ending idle connection to host localhost:57040 because the pool meets constraints; 1 connections to that host remain open
2017-12-03T21:03:10.823+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-03T21:03:10.824+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-03T21:12:09.497+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:12:09.498+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 1ms (3 connections now open to localhost:47017)
2017-12-03T21:13:09.497+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-03T21:23:09.513+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:23:09.515+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-03T21:24:09.514+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-03T21:32:29.525+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:32:29.527+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-03T21:33:29.526+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-03T21:34:09.531+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:34:09.533+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 2ms (3 connections now open to localhost:47017)
2017-12-03T21:35:09.532+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
2017-12-03T21:44:24.546+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:47017
2017-12-03T21:44:24.549+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:47017, took 3ms (3 connections now open to localhost:47017)
2017-12-03T21:45:24.547+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:47017 because the pool meets constraints; 2 connections to that host remain open
